docs/zz_generated.seed.yaml

```diff
@@ -274,6 +274,11 @@ spec:
         requests:
           cpu: 50m
           memory: 32Mi
+  # PauseProvisioning should only be used when migrating seed clusters, as it
+  # will make the Kubermatic Operator not reconcile the required components
+  # inside the seed cluster (like the seed-controller-manager, webhooks etc.).
+  # This setting does not "disable" a seed or the userclusters on it.
+  pauseProvisioning: false
```

Comment:
I was thinking that maybe having a label instead (e.g. `operator.k8c.io/skip-reconciling`) as proposed by @moelsayed would make more sense. The only use-case we have at the moment is this migration, I would avoid polluting the API with a field with a such reduced scope. @xrstf WDYT?

Reply:
Good idea. I'll change that.

Reply:
@irozzo-1A done